Source/WTF/wtf/PlatformUse.h

```diff
@@ -375,3 +375,8 @@
 #if PLATFORM(COCOA) && (HAVE(CGSTYLE_CREATE_SHADOW2) || HAVE(CGSTYLE_COLORMATRIX_BLUR))
 #define USE_GRAPHICS_CONTEXT_FILTERS 1
 #endif
+
+#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000) \
+    || ((PLATFORM(IOS) || PLATFORM(MACCATALYST)) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000)
```

Comment:
I am always confused by when to use `PLATFORM(IOS_FAMILY)` versus `PLATFORM(IOS)`. I believe that this rule means that USE_SANDBOX_VERSION_3 will be undefined for watchOS and tvOS. Is this your intention?

Reply:
> I am always confused by when to use `PLATFORM(IOS_FAMILY)` versus `PLATFORM(IOS)`. I believe that this rule means that USE_SANDBOX_VERSION_3 will be undefined for watchOS and tvOS. Is this your intention?

That is a good point. The intention is for also watchOS and tvOS to have version 3. I have removed this from the latest patch, and will enable in a separate patch.

Thanks for reviewing!